apb.yml

```diff
@@ -90,8 +123,48 @@ plans:
       - title: Storage Class
         name: storage_class
         type: string
-      - title: Virtual machine is imported as
-        name: image_type
-        default: vm
-        enum: ['vm', 'template']
+  - name: url-template
+    description: Create a virtual machine template from a downloaded disk image.
+    free: True
+    metadata:
+      displayName: c. Import as a template from URL.
+    parameters:
+      - title: OpenShift Admin Username
+        name: admin_user
```

Comment:
Is there a field for help text?  It would be nice to explain that administrator privileges are required to create a template that can be used by all users.

Reply:
No, I just double checked, just name and title.